controller/auth_utility_test.go

```diff
@@ -23,9 +23,9 @@ import (
 const (
 	// the various HTTP endpoints
 	endpointWorkItems              = "/api/workitems"
-	endpointWorkItemTypes          = "/api/workitemtypes"
+	endpointWorkItemTypes          = "/api/spaces/%s/workitemtypes"
 	endpointWorkItemLinkCategories = "/api/workitemlinkcategories"
-	endpointWorkItemLinkTypes      = "/api/workitemlinktypes"
+	endpointWorkItemLinkTypes      = "/api/spaces/%s/workitemlinktypes"
```

Comment:
Yes. it won't be a bad thing cause it is ugly to hardcode the URL for some actions.